docs/v0.6/en/getting-started/installation/greptimedb-dashboard.md

```diff
@@ -0,0 +1,12 @@
+# GreptimeDB Dashboard
+
+Visualization plays a crucial role in effectively utilizing time series data. To help users leverage the various features of GreptimeDB, Greptime offers a simple [dashboard](https://github.com/GreptimeTeam/dashboard).
+
+The Dashboard is embedded into GreptimeDB's binary since GreptimeDB v0.2.0. After starting [GreptimeDB Standalone](greptimedb-standalone.md) or [GreptimeDB Cluster](greptimedb-cluster.md), the dashboard can be accessed via the HTTP endpoint `http://localhost:4000/dashboard`. The current version of the dashboard supports [SQL queries](/user-guide/query-data/sql.md), [Python Scripts](/user-guide/python-scripts/overview.md), and [PromQL queries](/user-guide/query-data/promql.md).
```

Comment:
> The current version of the dashboard supports [SQL queries](/user-guide/query-data/sql.md), [Python Scripts](/user-guide/python-scripts/overview.md), and [PromQL queries](/user-guide/query-data/promql.md).

There is a potential that we forget to update this line when adding new features to dashboard. Can we find wording to describe it less likely to obsolete?